modules/filter/filter-popover.tsx

```diff
@@ -24,8 +26,12 @@ export function FilterPopover(props: Props): JSX.Element {
 			onRequestClose={() => onClosePopover(filter)}
 			placement={PopoverPlacement.BOTTOM}
 			popoverStyle={popoverContainer}
+			displayAreaInsets={safeareaInsets}
 		>
-			<FilterSection filter={filter} onChange={(filter) => setFilter(filter)} />
+			{/* This view wrapper shouldn't be needed but it does appear to fix a rendering issue */}
+			<View style={popoverContainer}>
```

Comment:
Is this view still needed in 5.1.7?

Reply:
Also: "+  29:4  warning  Props should be sorted alphabetically  react/jsx-sort-props"

Reply:
Still needed in 5.1.7. I'll sort the props.